shared/src/business/useCases/trialSessions/deleteTrialSessionInteractor.test.js

```diff
@@ -0,0 +1,164 @@
+const {
+  deleteTrialSessionInteractor,
+} = require('./deleteTrialSessionInteractor');
+const { User } = require('../../entities/User');
+
+const { MOCK_CASE } = require('../../../test/mockCase');
+
+const MOCK_TRIAL = {
+  caseOrder: [{ caseId: 'a54ba5a9-b37b-479d-9201-067ec6e335bb' }],
+  isCalendared: false,
+  judge: {
+    userId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+  },
+  maxCases: 100,
+  sessionType: 'Regular',
+  startDate: '2001-12-01T00:00:00.000Z',
+  term: 'Fall',
+  termYear: '2025',
+  trialLocation: 'Birmingham, AL',
+  trialSessionId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+};
+
+describe('deleteTrialSessionInteractor', () => {
+  let applicationContext;
+
+  it('throws error if user is unauthorized', async () => {
+    applicationContext = {
+      environment: { stage: 'local' },
+      getCurrentUser: () => {
+        return {
+          role: User.ROLES.petitioner,
+          userId: 'petitioner',
+        };
+      },
+      getPersistenceGateway: () => ({}),
+    };
+    await expect(
+      deleteTrialSessionInteractor({
+        applicationContext,
+        trialSessionId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+      }),
+    ).rejects.toThrow('Unauthorized');
+  });
+
+  it('throws an exception when it fails to find a trial session', async () => {
+    applicationContext = {
+      getCurrentUser: () => {
+        return new User({
+          name: 'Docket Clerk',
+          role: User.ROLES.docketClerk,
+          userId: '6805d1ab-18d0-43ec-bafb-654e83405416',
+        });
+      },
+      getPersistenceGateway: () => ({
+        getTrialSessionById: () => null,
+      }),
+    };
+
+    await expect(
+      deleteTrialSessionInteractor({
+        applicationContext,
+        trialSessionId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+      }),
+    ).rejects.toThrow('trial session not found');
+  });
+
+  it('throws error when trial session start date is past now', async () => {
+    applicationContext = {
+      getCurrentUser: () => {
+        return new User({
+          name: 'Docket Clerk',
+          role: User.ROLES.docketClerk,
+          userId: '6805d1ab-18d0-43ec-bafb-654e83405416',
+        });
+      },
+      getPersistenceGateway: () => ({
+        getTrialSessionById: () => ({
+          ...MOCK_TRIAL,
+        }),
+      }),
+      getUtilities: () => ({
+        createISODateString: () => '2050-12-01T00:00:00.000Z',
+      }),
+    };
+
+    await expect(
+      deleteTrialSessionInteractor({
+        applicationContext,
+        trialSessionId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+      }),
+    ).rejects.toThrow('Trial session cannot be updated after its start date');
+  });
+
+  it('throws error when trial session is calendared', async () => {
```

Comment:
```suggestion
  it('throws error if trial session is calendared', async () => {
```
I think this is a bit clearer.